Gemfile

```diff
@@ -38,7 +38,7 @@ group :development, :test do
   gem 'shoulda-matchers'
   gem 'launchy'
   gem 'faker'
-  gem 'factory_girl'
```

Comment:
We should just delete instead of commenting out.

Reply:
ya i think so too. I was running into factory girl issues so I switched the gem and configurations to factory_bot_rails, but no one else was having the same issues as me so we're going to see what works best for everyone before merging this